db/seeds.rb

```diff
@@ -1,5 +1,15 @@
+if !User.first
+  User.create!(email: Faker::Internet.email, password: 'jhkasdfhjkl', sign_in_count: 2)
+end
+
 if !Game.first
   10.times do
-    Game.create(name: Faker::Hipster.word.capitalize + ': the Game', min_player_count: Faker::Number.between(1, 2), max_player_count: Faker::Number.between(3,8), description: Faker::Hipster.sentences(4).join(' '))
+    Game.create!(name: Faker::Hipster.word.capitalize + ': the Game', min_player_count: Faker::Number.between(1, 2), max_player_count: Faker::Number.between(3,8), description: Faker::Hipster.sentences(4).join(' '))
+  end
+end
+
+if !Review.first
+  30.times do
+    Review.create!(rating: Faker::Number.between(1,5), body: Faker::Hipster.sentences(3).join(' '), game_id: Faker::Number.between(1,10), user_id: 1, victory_points: Faker::Number.between(1,100))
```

Comment:
this will not work as reliably as you might expect. You cannot guarantee that the `Game` objects created above will have ID's between 1 and 10. I would recommend using 

```
game: Game.order("RANDOM()").first
```

This will associate the review record with a random game.

you will also want to assign *user* to User.first, as you cannot assume that `User` created above will have an id of 1.